charts/openfga/values.yaml

```diff
@@ -39,6 +39,11 @@ service:
   type: ClusterIP
   port: 8080
 
+deployment:
+  extraEnvVars: []
```

Comment:
For consistency sake, please unnest these and colocate them along with the `podAnnotations` above

```
podAnnotations: {}

extraEnvVars: []
extraVolumes: []
extraVolumeMounts: []
```

Reply:
Thank you for the review.
I specified them only for the deployment manifest. Do you mean removing the `deployment` root key is the way to go?

If yes, what about the job templates (migration)? Should they also import these configurations? 

P.S The next thing that I would like to commit is the possibility to add only job annotations. In my scenario (using istio), the migration job is stuck on running state because the istio sidecar keeps it alive. So, I would like to exclude it via annotation.

Reply:
@jon-whit Did you have a time to review the above comment ?

Reply:
@karaimin sorry for the late response. Thanks for bumping this back up.

Keep the pod/deployment annotations at the top-level and we can be more specific for the migrate job later down the road. For now let's omit the extraEnvVars, extraVolumes, and extraVolumeMounts from the migrate job. Later, if we need them, we can introduce ones specific for that job (e.g. `migrate.extraEnvVars` etc..)

So for now the deployment annotations become top-level.



```

Reply:
For the follow up work we can introduce the top-level `migrate` section that you can provide `migrate.annotations` for, thus allowing you to exclude the job via annotations.

Reply:
Hi @jon-whit 
Sorry for my late response.
I applied the code review comments. Can you take a look ?